recipe/meta.yaml

```diff
@@ -13,10 +13,12 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
-  number: 0
+  number: 1
   script: 
     - python setup.py install --record=record.txt  # [not py3k]
     - python setup3.py install --record=record.txt  # [py3k]
+    - copy %PREFIX%\Lib\site-packages\pywin32_system32\*.dll %PREFIX%
```

Comment:
@ocefpaf not at all! I knew about `%LIBRARY_BIN%`, but got derailed because of the custom code in `pywintypes.py`. Should have figured out that having it on `%PATH%` is enough. 😁